cpp/include/cudf/quantiles.hpp

```diff
@@ -114,6 +118,7 @@ std::unique_ptr<table> quantiles(
  *
  * @param input           tdigest input data. One tdigest per row
  * @param percentiles     Desired percentiles in range [0, 1]
+ * @param stream CUDA stream used for device memory operations and kernel launches
  * @param mr              Device memory resource used to allocate the returned column's device
  * memory
```

Comment:
```suggestion
 * @param input One tdigest per row
 * @param percentiles Desired percentiles in range [0, 1]
 * @param stream CUDA stream used for device memory operations and kernel launches
 * @param mr Device memory resource used to allocate the returned column's device memory
```
Could you fix the formatting of all of these?